### ISSUE TITLE
Tool slot curve and consistency

### ISSUE THREAD

**Issue by jasonrohrer** [jasonrohrer/OneLifeData7#638]
Problems are described here:

http://onehouronelife.com/forums/viewtopic.php?pid=87848#p87848

**Comment by jasonrohrer** [comment]
More details in this post here:

http://onehouronelife.com/forums/viewtopic.php?pid=89828#p89773

**Comment by jasonrohrer** [comment]
More details in this post here:

http://onehouronelife.com/forums/viewtopic.php?pid=89984#p89984

**Comment by Tarr-OHOL** [comment]
Solution #1 to the tool slot issue: Tool pairings.

Goal: To create functional tool sets that compliment each other as a means of reducing traps, dysfunctional sets, and allowing players to do what they set out to do. 

Fishing rod + Shrimp net - "You have learned the fisherman tool set (Fishing rod + shrimp net)"
Knife + Sterilized knife - "You have learned the knife tool set (knife + sterilized knife)"
Steel axe + Bow saw - "You have learned the lumberjack tool set (Steel axe + Bone saw)"
Hot coals + Oven - "You have learned the cooking/chef tool set (Hot coals + Oven)"
Blowpipe + File - "You have learned the craftmans tool set (Blowpipe + File)"
Mallet + Stakes -"You have learned the construction tool set (Mallet + Stakes)
Adze + Shovel - "You have learned the minor construction tool set (Adze + Shovel)
Bow and Arrow + Fire-bow drill "You learned the survival basics tool set (Bow + Fire-bow drill)
Smithing hammer + Charcoal pencil "You learned the blacksmith tool set (Hammer + Pencil) 
Kiln + Forge *remove tongs as a tool, replace with forge* "You learned the artisan tool set (forge + kiln)
Knitting needles + Shears -"You have learned the shepherd skill (Knitting needles + shears)"
Car + Plane - "You have learned the travelers tool set (Car + Plane)
Horse/horse cart + Lasso "You have learned the cowboy tool set (Horse + Lasso) Yee haw.
Diesel water pump + Diesel mine "You have learned the diesel engine set (water pump + mine)
All newcomen tech should be its own thing (You have learned the newcomen suite - A, B, C, D, E)
Needle and Thread +Pads - "You have learned the medical tool set (Needle and thread + Pads.)"
Pickax + War sword - "You have learned the explorers tool set (Pickax + War sword)"
Camera + Loom - "You have learned the high class tool set (Camera + Loom)


This makes every tool functional with only one or two weird groupings (such as the last one.) Also, stone hatchet can either be removed or added to the axe grouping. 

Solution #2: Cleaning up and removing tools that are either too niche or broken. 

Items to remove under this solution:
Sterilized knife - It only works in one recipe as of currently and it's antivenom which is never used. 

Pads - You only learn pads when creating them instead of when treating people which means the tool slot exists for someone to make a bunch of pads instead of treating people. 

Stone hatchet - This tool exists as an Eve tax and isn't at all useful any other time besides the first 5 to ten minutes of a run. 

Tongs - These are a tax item as well. While it might make sense I'd rather see the logic come from knowing Kiln/Forge or Kiln + Hammer.

Kiln - If tongs are staying as a tool slot then kiln needs to be removed. Having both of these be tools is overkill especially since you can only learn this skill by covering a kiln with adobe. 

Lasso - This is only used for catching horses. If a horse is hitched to a post and you pick it up you don't actually learn the skill. One use tools = bad. 

Newcomens again should just be grouped - Newcomen drill has exactly one use, and most of the newcomen stuff is just for engines anyways. 

I'm sure more people can think of the taxing tools if asked. 


Solution #3: Ability to forget a tool every X years.

Goal: Allow players to keep useful skills while removing a useless one use skill from their list. 

Basically someone in the village is going to eventually have to learn the fire bow drill or newcomen drill. These are pretty useless tools 99% of the time (wheels aren't needed all the time and if you have to constantly restart the fire you've got big issues. Draw plate is the only item a newcomen drill makes.)

Other items in this category of "useless" Most newcomens will be used very few times in a life, swords can be learned on other players including allies which is 100% useless if done like this.

Stone hatchet also is a basically useless tool to have and a trap for people who don't know better.

So basically, every ten years or so allow a player to forget a tool to fix these sorts of issues.

**Comment by Tarr-OHOL** [comment]
Also, if the tool slots are left in their current iteration you really need to go through and fix all the flaws:

Sword is learned when recycled in a roller.
Healing someone with pads doesn't teach pads.
Sewing someone up doesn't teach sewing.

Weapons can be learned on ghosts, swords can be learned by attempting to attack a family member. 
Kiln is only learned while using adobe on it and not when lighting pottery.
Bow and arrow + Note doesn't require knowing how to use a bow. 
People in posses don't need to know weapon skills to attack.

Make it clearer to who knows what. Picking up items is fine until its both a game of finding the item AND finding the person. 

I can probably think/remember more but I'm a tad tired atm.

**Comment by jasonrohrer** [comment]
You've seen the thing where you get (+) above the head of each person around who knows that tool already, when you pick up an unlearned tool, right?

**Comment by Tarr-OHOL** [comment]
Yes, but it's still one of those things where you have to run around to do so which is why when I originally posted the pairs idea I added titles so you could always hover over someone to know what base skills they had. 

It's not the biggest gripe I have with the system just one of those things where you need to find whatever tool you're using then go around finding the guy who has the + too.
